pkg/apis/config/validation/account.go

```diff
@@ -31,6 +31,23 @@ func validateSubjects(subjects []rbac.Subject) field.ErrorList {
 	return allErrs
 }
 
+func verifySpace(account *configv1alpha1.Account) field.ErrorList {
+	space := configv1alpha1.AccountSpace{}
+	err := convert.ObjectToObject(account.Spec.Space, &space)
```

Comment:
I don't think this is necessary here, it is fine to just reuse the account.Spec.Space